migtests/scripts/postgresql/create_pg_user

```diff
@@ -3,7 +3,7 @@ source migtests/scripts/postgresql/env.sh
 source migtests/scripts/functions.sh
 
 cat > pg-create-user.sql << EOF
-SELECT 'CREATE USER ${SOURCE_DB_USER} PASSWORD ''${SOURCE_DB_PASSWORD}'''
+SELECT 'CREATE USER ${SOURCE_DB_USER} WITH SUPERUSER PASSWORD ''${SOURCE_DB_PASSWORD}'''
```

Comment:
I guess we shouldn't be commiting this? :D

Reply:
Yup! Thanks. Let me see what I can do to separate the offline and live migration user/grants creation. We'll figure out the exact steps later.